src/scripts.js

```diff
@@ -8,20 +9,28 @@ const userStepGoal = document.querySelector('.user-step-goal');
 const userGoalAverage = document.querySelector('.step-goal-average');
 const friendsList = document.querySelector('.friends-list');
 const stepComparison = document.querySelector('.step-goal-comparison')
+const dayHydrationBox = document.querySelector('.hydration-day')
+const weekHydradtionBox = document.querySelector('.hydration-week')
 
-function populateUserInfo(id) {
-  let user = new User(userRepo.getUserData(id))
+function populateUserInfo() {
+  let user = new User(userRepo.getUserData(42));
```

Comment:
Why did you get rid of the id parameter?

Reply:
Ah was playing around with stuff cuz something was going on. Just forgot to switch back